tests/fixtures/exception_groups.py

```diff
@@ -0,0 +1,48 @@
+# flake8: noqa
+
+# this creates an exception with a very small __traceback__, so it's easier to
+# assert against in tests
+def generate_exception(exception_class, message):
+    try:
+        raise exception_class(message)
+    except BaseException as exception:
+        return exception
+
+
+def raise_exception_group_with_no_cause():
+    raise ExceptionGroup(
+        'the message of the group',
+        [
+            generate_exception(Exception, 'exception #1'),
+            generate_exception(ArithmeticError, 'exception #2'),
+            generate_exception(NameError, 'exception #3'),
+            generate_exception(AssertionError, 'exception #4'),
```

Comment:
Worth adding a nested ExceptionGroup here so you can verify what is, and isn't, reported from it?